mongorm/fields/SafeDictField.py

```diff
@@ -9,7 +13,7 @@ def deepCoded( dictionary, coder ):
 	toCode = deque( [dictionary] )
 	while toCode:
 		nextDictionary = toCode.popleft( )
-		for key, value in nextDictionary.items( ): # can't be iteritems as we're changing the dict
+		for key, value in list(nextDictionary.items( )): # can't be iteritems as we're changing the dict
```

Comment:
I just tried it without the `list` and it works, so perhaps you don't need it 😁

Reply:
The difference is because in python2 items() would return you a copy of the key/value pairs in `nextDictionary`, but in python3 `items()`, is now a iterator. I think you need `list` because in the loop it does a `del nextDictionary[key]` which modifies the `dict`, which will cause problems with your iterator.